.github/workflows/aws_dbfn_deploy.yml

```diff
@@ -0,0 +1,35 @@
+name: Upload Init Fn
```

Comment:
Is there a reason why you made this job in a file that's separate from `build.yml`? (To be clear I think it's a good idea to separate AWS deployment stuff from normal build stuff, though I'd name the file to be something a lot terser while changing `name: Upload Init Fn` to something more AWS-specific.)

Reply:
I initially tried to implement it as additional steps on the release job but the permissions thing made them incompatible.  Currently it is completely orthogonal but it *could* be implemented as a subsequent job in `build.yaml` depending on `release`.

Reply:
That might be a good idea - it might make sense that you only upload the init fn if the build is successful, for example.

Reply:
Yeah I also think it would be better that way. I'll move it over and cut yet another test release